includes/sanitizers/class-amp-style-sanitizer.php

```diff
@@ -1456,41 +1507,64 @@ function( $selector ) {
 					$selectors   = explode( $between_selectors . ',', $split_stylesheet[ ++$i ] );
 					$declaration = $split_stylesheet[ ++$i ];
 
+					// @todo The following logic could be made much more robust of PHP-CSS-Parser did parsing of selectors. See <https://github.com/sabberworm/PHP-CSS-Parser/pull/138#issuecomment-418193262>.
```

Comment:
nit pick: "more robust *if* PHP-CSS-Parser did.."

Reply:
Fixed in ccf80c22e4d8984da4cf64fbff23a973a89996d4